tasks/collect-cosign-params/README.md

```diff
@@ -7,3 +7,6 @@ Tekton task that collects cosign options from the data file.
 | Name      | Description                                                             | Optional | Default value  |
 |-----------|-------------------------------------------------------------------------|----------|----------------|
 | dataPath  | Path to the JSON string of the merged data to use in the data workspace | No       | -              |
+
+## Changes in 0.1.1
+  * Fix linting issues in this task
```

Comment:
I would prefer no leading space here - there is no reason for it. I know we're a bit inconsistent in the repo with this, but most READMEs don't have any space here I think. 

But changing it would invalidate your e2e run, right? So you don't want that, I'm fine with leaving it.